dev-tools/mage/target/update/update.go

```diff
@@ -0,0 +1,25 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package update
+
+import "github.com/magefile/mage/sh"
+
+// Update updates the generated files (aka make update).
+func Update() error {
+	return sh.Run("make", "update")
+}
```

Comment:
At least metricbeat, journalbeat and heatbeat also have this implementation, do you plan to import this target in these magefiles?

I tried to use mage targets to replace update in metricbeat and [I think](https://github.com/elastic/beats/blob/ab996ce2d36555b21a7b22a523500ada1d9e6d9a/metricbeat/magefile.go#L113) that we were only missing the generation of fields documentation, it'd be good to migrate this part so we can use only mage for this.

Reply:
Yah, the "step two" of this is to start cleaning up the non-generator magefiles. Actually cleaning up the targets themselves is a different matter, as I kinda trying to devote my time to getting rid of the python 2 stuff first.